Header header
ObjectInfo[] object_info
Relation[] relations
